packages/core/docs/commercetools/configuration.md

```diff
@@ -18,7 +18,58 @@ Commercetools configuration is located in two places:
 }]
 ```
 
-- `useNuxtI18nConfig` - when this property is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
+- `useNuxtI18nConfig` - when enabled, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. Otherwise, the `i18n` config should be passed directly to this package configuration. You can read more about it on [Internationalization](../advanced/internationalization.md) page. There are things specific to commercetools, which you can read more about below.
+
+### Adding available states to the country in Checkout and MyAccount
+In order to add a new state, add `states` field in the desired country inside `i18n.countries` in `nuxt.config.js`:
+```js
+i18n: {
+  countries: [
+    { name: 'US',
+      label: 'United States',
+      states: [
+        'California',
+        'Nevada'
+      ]
+    },
+    // ...
+  ]
+}
+```
+::: warning Each state requires the tax rules
+A state without tax rules cannot work properly. Make sure to add the tax rules for each state in the commercetools.
+:::
+
+:::details How to add tax rules to the state?
+In commercetools' dashboard, open *Settings* -> *Project settings* -> *Taxes* and click **Add rate** button to add tax rates for new states:
```

Comment:
just put the link to the docs, don't create tutorial in our docs for 3rd party service